spinflip/spin_flip_tools.py

```diff
@@ -87,6 +87,48 @@ def sigma(flux):
 #data_loc is the spinflip file to compute
 #merger_data_loc is the merger grid data location
 #location is where in the merger data to evaluate stuff like ye, rho (=[x,y,z])
+class Gradients:
+    def __init__(self, gradient_filename, merger_data_loc):
+        
+        #allgrad object (dims: spacetime up, spacetime (gradient) low, x,y,z, F, F)
+        self.gradJ, self.x, self.y, self.z, self.it, self.limits = read_gradients(gradient_filename)
+
+        
+        # S_R/L_nu gradient (spacetime up, spacetime (gradient) low, x,y,z, F, F) [tranpose gradJ so new lower index is last and the sigma function works properly, then transpose back]
+        self.grad_S_R_nu, self.grad_S_L_nu = sigma(np.transpose(self.gradJ, axes = (0,5,6,1,2,3,4)))
+        self.grad_S_R_nu, self.grad_S_L_nu = np.transpose(self.grad_S_R_nu, axes = (0,3,4,5,6,1,2)), np.transpose(self.grad_S_L_nu, axes = (0,3,4,5,6,1,2))        
+        
+        #THIS NEEDS GR TREATMENT
```

Comment:
This should be deleted in favor of the GR gradients implemented in the previous PR